### DISCUSSION TITLE
Turn off snippet jump points

### DISCUSSION THREAD

**Discussion by dotzenith** [saghen/blink.cmp#470] [Q&A] [answered]
is there a way to turn off jump points for snippets? (i.e. no need to use `snippet_forward` and `snippet_backward`) and just leave arguments empty?

Especially in the case of methods and functions that are *not* snippets, it almost feels like extra work in case of functions that really just take one argument.

Apologies if I missed a config option for this

**Comment by Parsifa1** [comment]
maybe you need:
```lua
capabilities.textDocument.completion.completionItem.snippetSupport = false
```
set it for your lspserver capabilities

**Reply by dotzenith** [reply]
Doesn't seem to work for me

**Reply by telemachus** [reply, marked as answer]
This may or may not help, but I had the same problem, and it turns out that not every language server handles turning off snippets in the same way. For the Lua language server, for example, this works for me (as opposed to the standard that Parsifa1 mentions above).

```lua
lspconfig.lua_ls.setup({
    settings = {
        Lua = {
            -- There are other settings, but the following disables snippets.
            completion = {
                callSnippet = "Disable",
                keywordSnippet = "Disable",
            },
        },
    },
})
```

**Reply by dotzenith** [reply]
Thank you @telemachus, that was indeed the right direction. If anyone comes looking for this afterwards, this is what I had to do for rust:

```lua
settings = {
	["rust-analyzer"] = {
		completion = {
			callable = {
				snippets = "add_parentheses", -- Instead of the default "fill_arguments"
			},
		},
	},
}
```

**Reply by Parsifa1** [reply]
LGTM!
